bliss/encoder.py

```diff
@@ -139,6 +139,8 @@ def get_input_tensor(self, batch):
                 input transformations to use
         """
         # If using five-band cached simulator, only select bands encoder is expecting
+        if batch["images"].shape[1] < self.n_bands:
+            warnings.warn("Encoder is expecting additional bands from input.")
```

Comment:
This should've been added in the multi-band PR - just raising a warning in case the user forgot to set the `z_score` input transform parameter to True when performing multi-band inference.